### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by bartaz**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/request-a-missing-resource-or-asset.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+name: Request a missing resource or asset
+about: Let us know if you can't find a design or brand asset you are looking for
+title: ''
+labels: Resource request
+assignees: ''
+
+---
+
+**What kind of resource/asset you are looking for? (logo, image, icon, banner, etc)**
```
Comment (edlerd) [inline .github/ISSUE_TEMPLATE/request-a-missing-resource-or-asset.md:10]: In the heading I'd phrase it "What kind of resource/asset are you looking for?"

**Review by edlerd — APPROVED** [review]
LGTM, just one suggestion.

File: .github/ISSUE_TEMPLATE/request-a-missing-resource-or-asset.md (hunk 1 above)
Comment (bartaz) [reply .github/ISSUE_TEMPLATE/request-a-missing-resource-or-asset.md:10]:
```suggestion
**What kind of resource or asset are you looking for?**
```

File: .github/ISSUE_TEMPLATE/request-a-missing-resource-or-asset.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+Tell us what kind of resource you are looking for …
```
Comment (bartaz) [inline .github/ISSUE_TEMPLATE/request-a-missing-resource-or-asset.md:12]:
```suggestion
Tell us what kind of resource you are looking for (logo, image, icon, banner, etc) …
```

**Pull request merged**